gds-api-adapters.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 3.1"
   s.files        = Dir.glob("lib/**/*") + Dir.glob("test/fixtures/**/*") + %w[README.md Rakefile]
   s.require_path = "lib"
+  s.add_dependency "activesupport"
```

Comment:
Are there any users of this gem that don't already have ActiveSupport?

I do get that it's still a bit gross to do this even if that's currently the case, but if it's not the case I'd be a lot less keen on doing this 😆 

Maybe there are standalone gems that implement these methods?